devservices/ecr/src/main/java/io/quarkus/amazon/devservices/ecr/EcrDevServicesProcessor.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.amazon.devservices.ecr;
+
+import org.testcontainers.containers.localstack.LocalStackContainer.Service;
+
+import io.quarkus.amazon.common.deployment.spi.AbstractDevServicesLocalStackProcessor;
+import io.quarkus.amazon.common.deployment.spi.DevServicesLocalStackProviderBuildItem;
+import io.quarkus.amazon.ecr.runtime.EcrBuildTimeConfig;
+import io.quarkus.deployment.annotations.BuildStep;
+
+public class EcrDevServicesProcessor extends AbstractDevServicesLocalStackProcessor {
+
+    @BuildStep
+    DevServicesLocalStackProviderBuildItem setupSts(EcrBuildTimeConfig clientBuildTimeConfig) {
+        return this.setup(Service.STS, clientBuildTimeConfig.devservices());
```

Comment:
Have you had a chance to test with the pro image of LocalStack? I saw that it also interacts with KMS, and I want to ensure it will work since I had an issue with SSM requiring events to be enabled as well.

```suggestion
        return this.setup(EnabledService.named("ecr"), clientBuildTimeConfig.devservices());
```